docs/integration.md

```diff
@@ -51,21 +51,23 @@ Note: Need to use **@import CleverTapSDK;** instead of **#import <CleverTap-iOS-
 	    // Register the CleverTap ActivityLifecycleCallback; before calling super
         ActivityLifecycleCallback.register(this);	
         super.onCreate();
+        // Init CleverTapSDK React Native integration
+        CleverTapRnAPI.initReactNativeIntegration(this);
     }
     ```
 
-3. Optionally Override onCreate in MainActivity.java to notify CleverTap of a launch deep link  (`android/app/src/[...]/MainActivity.java`)
+3. Optionally override onCreate in MainActivity.java to notify CleverTap of a launch deep link  (`android/app/src/[...]/MainActivity.java`)
     ```java
     import com.clevertap.react.CleverTapModule;
     import android.os.Bundle;
     
     public class MainActivity extends ReactActivity {
-		// ...
+        // ...
 
-		@Override
-   		protected void onCreate(Bundle savedInstanceState) {
-        	super.onCreate(savedInstanceState);
-        	CleverTapModule.setInitialUri(getIntent().getData());
+        @Override
+        protected void onCreate(Bundle savedInstanceState) {
+            super.onCreate(savedInstanceState);
+            CleverTapRnAPI.notifyLaunchDeepLink(getIntent().getData());
     	}
```

Comment:
Will it be better if we keep the name of the API same as before? @vasct

Reply:
Probably yes, I didn't like the name, but it seems there is also getInitialUrl, which is more consistent with the previous name.